src/snmp.c

```diff
@@ -1264,12 +1264,17 @@ static int csnmp_strvbcopy(char *dst, /* {{{ */
     src = (char *)vb->val.string;
   else if (vb->type == ASN_BIT_STR)
     src = (char *)vb->val.bitstring;
-  else if (vb->type == ASN_IPADDRESS) {
```

Comment:
Don't remove the braces, yes there is only one statement, but it is multiline...
And also add braces to the below hunks too